packages/mad_pay_platform_interface/lib/src/models/apple_parameters.dart

```diff
@@ -54,7 +54,8 @@ extension AppleParametersX on AppleParameters {
   /// Mapped this model to proto model
   proto.AppleParameters get toProto => proto.AppleParameters(
         merchantIdentifier: merchantIdentifier,
-        merchantCapabilities: merchantCapabilities?.toProto,
+        merchantCapabilities:
+            merchantCapabilities?.map((MerchantCapabilities e) => e.toProto!),
```

Comment:
You can get rid of the bang operator here if you modify toProto:
```
/// MerchantCapabilities extension
extension MerchantCapabilitiesX on MerchantCapabilities {
  /// Mapped this model to proto model
  proto.MerchantCapabilities get toProto {
    switch (this) {
      case MerchantCapabilities.threeds:
        return proto.MerchantCapabilities.THREEDS;
      case MerchantCapabilities.credit:
        return proto.MerchantCapabilities.CREDIT;
      case MerchantCapabilities.debit:
        return proto.MerchantCapabilities.DEBIT;
      case MerchantCapabilities.emv:
        return proto.MerchantCapabilities.EMV;
    }
  }
}
```

You can also fix ShippingType:

```
/// ShippingType extension
extension ShippingTypeX on ShippingType {
  /// Mapped this model to proto model
  proto.ShippingType get toProto {
    switch (this) {
      case ShippingType.shipping:
        return proto.ShippingType.SHIPPING;
      case ShippingType.delivery:
        return proto.ShippingType.DELIVERY;
      case ShippingType.servicePickup:
        return proto.ShippingType.SERVICE_PICKUP;
      case ShippingType.storePickup:
        return proto.ShippingType.STORE_PICKUP;
    }
  }
}
```